diff --git a/workers/lib/middlewares/auth/jwt.guard.js b/workers/lib/middlewares/auth/jwt.guard.js
index 9c8c038..429d70d 100644
--- a/workers/lib/middlewares/auth/jwt.guard.js
+++ b/workers/lib/middlewares/auth/jwt.guard.js
@@ -41,8 +41,8 @@ class JwtGuard extends BaseGuard {
     }
   }
 
-  _parsePayload (payload, req) {
-    return { id: payload.userId || req.params?.userId || req.query?.userId || req.body?.userId }
+  _parsePayload (payload) {
+    return { id: payload.userId }
   }
 
   /**
